db/schema.rb

```diff
@@ -318,6 +318,7 @@
     t.boolean "ready_for_status_check", default: false
     t.datetime "notification_sent_at", comment: "The time a notification was sent"
     t.datetime "last_batch_claimed_at"
+    t.string "sponsor_id"
```

Comment:
Request: could you please add a comment to provide more information about this new field in the database?  For example, the comment could indicate that sponsor id can be evaluated to determine if an enrollment is enhanced IPP.  Thanks!

Reply:
Thanks for adding the comment. 👍🏻